src/reporter/reporter.py

```diff
@@ -192,7 +192,7 @@ def notify():
         msg = "Notification not processed or not updated"
```

Comment:
can we log with error level the payload?

Reply:
> can we log with error level the payload?

 @chicco785 , As per my understanding we need to add `payload` with error msg `"Notification not processed or not updated"`. Please confirm.